json_newline_writer_sink.go

```diff
@@ -0,0 +1,36 @@
+package health
+
+import "io"
+
+type JsonNewlineWriterSink struct {
+	jj *JsonWriterSink
```

Comment:
@taylorchu so there is a problem with that since they have the same interface

so JsonNewlineWriterSink.EventFoo needs to go to JsonWriterSink, and that cant happen if its embedded